content/04.study.md

```diff
@@ -347,9 +347,9 @@ A field poised for dramatic revolution by deep learning is bioimage analysis.
 Thus far, the primary use of deep learning for biological images has been for segmentation---that is, for the identification of biologically relevant structures in images such as nuclei, infected cells, or vasculature---in fluorescence or even brightfield channels [@doi:10.1371/journal.pcbi.1005177].
 Once so-called regions of interest have been identified, it is often straightforward to measure biological properties of interest, such as fluorescence intensities, textures, and sizes.
 Given the dramatic successes of deep learning in biological imaging, we simply refer to articles that review recent advancements [@doi:10.3109/10409238.2015.1135868; @doi:10.1371/journal.pcbi.1005177; @doi:10.1007/978-3-319-24574-4_28].
-For deep learning to become commonplace for biological image segmentation, however, user-friendly tools must be developed.
+For deep learning to become commonplace for biological image segmentation, we need user-friendly tools.
```

Comment:
Do we want to preserve this change? I trust your call on reverting the reference below.

Reply:
I don't mind either way, I suppose keeping the edit would be nice (if you can make that change yourself? Not sure how I'd do it, sadly!